### DISCUSSION TITLE
Hola buenos dias , desde España ,YO he sido estafado por virgobanc.net

### DISCUSSION THREAD

**Discussion by mawelis** [community/community#114463] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hola me llamo Salvador Moreno Ledesma , vivo en España,y he sido estafado por , virgobanc.net, He mirado en la pagina y me ha traido hasta esta comunidad, he entrado en Impector , https://virgobanc.net/platform.html#/account/withdrawal/request, Estos maechores no me dejan sacar mis ganancias , despues de haber invertido 70,000 mil euros  ¿ No se si me pueden ayudar ??

**Comment by Akash1134** [comment]
Hi @mawelis , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
